src/functions.c

```diff
@@ -253,7 +253,8 @@ static void libraryUnlink(librariesCtx *lib_ctx, libraryInfo* li) {
 
 static int libraryLink(librariesCtx *lib_ctx, libraryInfo* li, sds* err) {
     int ret = C_ERR;
-    dictIterator *iter = dictGetIterator(li->functions);
+    dictIterator *iter = NULL;
+    iter = dictGetIterator(li->functions);
     dictEntry *entry = NULL;
```

Comment:
i see you have a few more of these redundant lines.

Reply:
This is the variable definition, you want to not initialised it either?

Reply:
ohh, sorry, i missed the fact it creates the variable here.
ignore me.